src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -767,6 +814,9 @@ private void checkParamTags(final List<JavadocTag> tags,
         final List<DetailAST> params = getParameters(parent);
         final List<DetailAST> typeParams = CheckUtil
                 .getTypeParameters(parent);
+        // Validating the order of these parameters
+        checkParamTagsOrder(new ArrayList<>(tags), new ArrayList<>(params),
+                new ArrayList<>(typeParams));
```

Comment:
is there a reason we can't send the list as is without re-wrapping around a new instance?

Reply:
Because we are deleting the parameters from the list. Deletion changes the order of the parameters. Hence we created a copy of the original list, so it maintains the original order of all the parameters.